src/index.js

```diff
@@ -1,9 +1,16 @@
 import React, {PropTypes as T} from 'react';
-import { soundManager } from 'soundmanager2';
 
 const pendingCalls = [];
 let initialized = false;
 
+if (typeof window !== 'undefined') {
+  var soundManager = require('soundmanager2');
```

Comment:
This works because `var` is function scoped, but it's confusing this way. Let's use a `let` and define the variable outside the `if`, like this:

```js
let soundManager;

if (typeof window !== 'undefined') {
  soundManager = require('soundmanager2');
}
```

I think it's clearer this way